companies/zucchetti.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "zucchetti",
+    "relevant-countries": [
```

Comment:
Please make sure that these are correct ISO 3166-1 alpha-2 country codes, otherwise the schema validation will fail.